SPECS/kernel/config

```diff
@@ -7318,6 +7318,9 @@ CONFIG_ENCRYPTED_KEYS=y
 # CONFIG_USER_DECRYPTED_DATA is not set
 # CONFIG_KEY_DH_OPERATIONS is not set
 CONFIG_SECURITY_DMESG_RESTRICT=y
+CONFIG_PROC_MEM_ALWAYS_FORCE=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/8552508033b26803c673598c925257a5ea6f4d98#diff-c597b2e78c72d7e7b2379ebab7a7d188d0bf67058276659240545d0a9b43548eR50 

The current default settings (as set here) maintain the traditional behavior. Keeping traditional behavior for now as there are concerns this could break gdb and other processes which need to affect the memory map of a process (via ptrace).